packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -37,14 +39,13 @@ import {
 import * as languageServer from './languageServer';
 import { nls } from './messages';
 import { telemetryService } from './telemetry';
-import { ApexTestOutlineProvider } from './views/testOutlineProvider';
+import { testOutlineProvider } from './views/testOutlineProvider';
```

Comment:
An instance of ApexTestOutlineProvider is now exposed (testOutlineProvider which is a quasi-singleton).  I spoke with Ananya about this and change the code to use the instance that's exported.